marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -571,25 +571,19 @@ const GraphVis: React.FC<Props> = (props) => {
           setCoords(newCoords);
           props.saveEntityCoords(nodes[0], positions.x, positions.y);
         }
+      } else {
+        if (props.entityTypes) {
+          let updatedEntityDefinitions: any = [];
+          props.entityTypes.forEach(ent => {
+            let entityDefinition: EntityModified = getDefinitionsPayload(ent, {graphX: coords[ent.entityName].graphX + event.event.deltaX, graphY: coords[ent.entityName].graphY + event.event.deltaY});
```

Comment:
@AkshayBajajML - Just wondering if **coords** will always get the updated coordinates at new drag location, without using network.getPositions(). Please confirm this.

Reply:
@xnikhil08 in my investigation, getPositions() did not return updated coordinates when panning (unlike when an individual mode is moved). That's why we have to use the delta values (plus the previous positions as saved in the coords) to get the updated coords to be saved during a pan.